cate/ds/esa_cci_odp.py

```diff
@@ -1441,7 +1441,8 @@ async def _init_file_list(self, monitor: Monitor = Monitor.NONE):
         # Compute the data source's temporal coverage
         for file_rec in file_list:
             if file_rec[1]:
-                file_start_date = datetime.strptime(file_rec[1].split('.')[0], _TIMESTAMP_FORMAT)
+                file_start_date = datetime.strptime(file_rec[1].split('/')[0].split('.')[0].split('+')[0],
+                                                    _TIMESTAMP_FORMAT)
                 file_end_date = file_start_date + time_delta
```

Comment:
This may be incorrect, because if the `datetime` is a time range (= has "/" in it) then "<file_start_date>/<file_end_date>", so `file_end_date` is already determined exactly.